src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,120 +33,188 @@ redirects:
+        ### Set app permissions 
+
+Once you've applied the plugins, you need to set your system permissions so your app can make requests to `INTERNET` and `ACCESS_NETWORK_STATE`. Add this code snippet to your `AndroidManifest.xml` file:
+
+  ```xml
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   ```
-  import com.newrelic.agent.android.NewRelic;
-  ```
+        
+    </Step>
+    <Step>
+        ### Ensure the Android agent runs when your app runs
 
-  <Callout variant="important">
-    We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
-  </Callout>
-5. To initialize mobile monitoring: In the `onCreate()` method, add this call:
+From your app's manifest, find the activity subelement. You need to import the `NewRelic` class in your `onCreate()` method. This initializes mobile monitoring when your app starts up: 
 
-   ```java
-   NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
-   ```
+```xml
+import com.newrelic.agent.android.NewRelic;
+```
+
+    </Step>
+    <Step>
+    ### See your data in New Relic
 
-6. Optional: Change the [logging level](/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags#logging).
 7. To build and run your application: Clean your project, then run your app in an emulator or device to generate traffic.
 8. Wait a few minutes, then view data for your Android app from the [**Summary** page](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-overview-page): Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app)**.
 
-If you have problems with your Android installation, or if you do not see data in the mobile monitoring UI, follow the [troubleshooting procedures](/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/no-data-appears-android).
-
-<InstallFeedback />
+[troubleshooting procedures](/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/no-data-appears-android)
 
-## Update your Android installation [#upgrading]
+    </Step>
+</Steps>
 
-To keep your Android agent up to date, follow the procedures to [upgrade the Android agent SDK](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/upgrading-new-relic-mobiles-android-sdk) for New Relic. To install the latest Android version for your mobile app: Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+For details on how to configure the behavior of the mobile monitoring agent plugin during Gradle builds, see the [New Relic Gradle plugin](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-new-relic-gradle-plugin) extension guide.
 
 ## Android 4.x: Multidex support [#4x-multidex]
```

Comment:
This section reads like a mix between troubleshooting and reference material rather than a necessary set up step for multidex users. Should we remove this section and put it in troubleshooting?